### PR TITLE
CNAPP-10211-IntegrationEntity

### PR DESCRIPTION
**Pull request by chkp-yogevk**

### CODE REVIEW

File: dome9/resource_dome9_integration.go [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+	postModel := integrations.IntegrationPostRequestModel{
+		Name:          d.Get("name").(string),
+		Type:          integrations.IntegrationType(d.Get("type").(string)),
+		Configuration: []byte(d.Get("configuration").(string)),
+	}
+
+	return postModel, nil
+}
+
+// CRUD API
```
Comment (chkp-dolevh) [inline dome9/resource_dome9_integration.go]: pls remove the comment

File: dome9/resource_dome9_integration.go [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+	resp, _, err := d9Client.integration.GetById(d.Id())
+	if err != nil {
+		return err
+	}
+
+	d.SetId(resp.Id)
+	_ = d.Set("name", resp.Name)
+	_ = d.Set("type", resp.Type)
+	// _ = d.Set("configuration", string(resp.Configuration))
```
Comment (chkp-dolevh) [inline dome9/resource_dome9_integration.go]: pls remove

File: website/docs/d/integration.html.markdown [hunk 3]
```diff
@@ -0,0 +1,44 @@
+﻿---
+layout: "dome9"
+page_title: "Check Point CloudGuard: dome9_notification"
+sidebar_current: "docs-data-source-dome9-notification"
+description: |- 
+  Fetches details of a specific CloudGuard integration.
+---
+
+# dome9\_notification
```
Comment (chkp-dolevh) [inline website/docs/d/integration.html.markdown]: pls change it to:
# Data Source: dome9_notification

File: website/docs/d/integration.html.markdown [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+- `id` - (Required) The ID of the CloudGuard integration to retrieve information for.
+
+## Attribute Reference
+
+The following attributes are exported:
+
+- `name` - The name of the CloudGuard integration.
+- `type` - The type of the CloudGuard integration.
+- `configuration` - The configuration details of the CloudGuard integration.
```
Comment (chkp-dolevh) [inline website/docs/d/integration.html.markdown]: pls explain more about what "configuration" is

File: website/docs/r/integration.html.markdown [hunk 5]
```diff
@@ -0,0 +1,63 @@
+﻿---
+layout: "dome9"
+page_title: "Check Point CloudGuard: dome9_integration"
+sidebar_current: "docs-resource-dome9-integration"
+description: |- 
+  Creates and manages CloudGuard integrations.
+---
+
+# dome9\_integration
```
Comment (chkp-dolevh) [inline website/docs/r/integration.html.markdown]: pls change to:
# dome9_integration

File: website/docs/r/continuous_compliance_notification.html.markdown [hunk 6]
```diff
@@ -7,6 +7,10 @@ description: |- Creates continuous compliance notification in Dome9
 
 # dome9_continuous_compliance_notification
 
+
+**Notice: This resource is obsolete due to changes in the lifecycle of the API. It will be partially supported until its end of life on January 1, 2025.
+The functionality for this resource has been replaced and the new implementation for this resource is `dome9_notification`[Learn more](https://registry.terraform.io/providers/dome9/dome9/latest/docs/data-sources/notification).**
```
Comment (chkp-dolevh) [inline website/docs/r/continuous_compliance_notification.html.markdown:13]: Please add an explanation of how to migrate the old dome9_continuous_compliance_notification to the new one

File: website/docs/r/integration.html.markdown (hunk 5 above)
Comment (chkp-yogevk) [reply website/docs/r/integration.html.markdown]: this is like it right now :)

**Review by chkp-dolevh — APPROVED** [review]
Pls check the UT

**Pull request merged**